vscode/src/toitExec.ts

```diff
@@ -41,11 +41,8 @@ async function executeRunCommand(ctx: Context, device?: Device) {
   try {
     if (!device) device = await selectDevice(ctx, { "activeOnly": true, "simulatorOnly": false });
 
-    const commandProcess = cp.spawn(ctx.toitExec, [ "dev", "-d", device.name, "run", filePath ]);
-    const toitOutput: OutputChannel = ctx.outputChannel(device.deviceID, device.name);
-    toitOutput.show(true);
-    commandProcess.stdout.on("data", data => toitOutput.append(`${data}`));
-    commandProcess.stderr.on("data", data => toitOutput.append(`${data}`));
+    cp.spawn(ctx.toitExec, [ "dev", "-d", device.name, "run", filePath ]);
+    ctx.startDeviceOutput(device);
```

Comment:
Even if we assume that the `logs` output is good enough, we now have a race condition: if the `run` is too fast, the logs won't show the output of the run command anymore, since the `startDeviceOutput` would be too late.